tests/CMakeLists.txt

```diff
@@ -59,6 +53,7 @@ set(TEST_SOURCES httpfake.h test_utils.cc test_utils.h uptane_vector_tests.cc)
 include(CMakeParseArguments)
 
 add_library(testutilities test_utils.cc)
+list(APPEND TEST_LIBS testutilities)
```

Comment:
Do `gtest` and `gmock` still get linked in somehow?

Reply:
`TEST_LIBS` is defined in the parent scope here: https://github.com/advancedtelematic/aktualizr/blob/8b94dbd8957b99065b473794ca1a86203aa76df8/CMakeLists.txt#L393, that's how.

Reply:
Ah, thanks, didn't realize it was redundant.